.github/workflows/ci.yml

```diff
@@ -14,6 +14,26 @@ jobs:
   # Note that vcpkg dependencies takes the majority of the build time.
   # We cache them using GitHub Actions cache and export, making the scripts below a bit more complex.
 
+  # Enforces the consistency of code formatting using .editorconfig and the dotnet-format tool.
+  check-format:
+    if: github.event_name == 'push' || github.event.pull_request.head.repo.id != github.event.pull_request.base.repo.id
+    runs-on: ubuntu-18.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Setup .NET Core SDK
+        uses: actions/setup-dotnet@v1
+        with:
+          dotnet-version: ${{ env.dotnet_3_version }}
+      - name: Restore tool
+        run: dotnet tool restore
+      - name: Check format - Library
+        run: dotnet format "./csharp/ParquetSharp.csproj" --check
```

Comment:
We could simplify those by pointing to the project directory, i.e. `dotnet format csharp --check`

Reply:
nicee! I missed that, pushing it now